### PR TITLE
from bnjul

### PR DESCRIPTION
**Pull request by bnjul**

### CODE REVIEW

**Comment by AyushThapaMagar** [comment]
ramro xaina

**Review by abhidahal — COMMENTED** [review]
This is good but we need C++ code

File: bnjul.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#include<stdio.h>
+void main ()
+{
```
Comment (lukesholi) [inline bnjul.txt:3]: nice

**Review by abhidahal — APPROVED** [review]
lgtm

File: bnjul.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+printf("Hellow daai";
```
Comment (abhidahal) [inline bnjul.txt]: missing closing bracket.

**Review by abhidahal — CHANGES_REQUESTED** [review]
lgtm

**Comment by PrashantShr-22** [comment]
fanta stick

File: bnjul.txt (hunk 1 above)
Comment (jungprajwol) [reply bnjul.txt:3]: gantey

Comment (jungprajwol) [reply bnjul.txt:3]: chyantey

Comment (jungprajwol) [reply bnjul.txt:3]: alik ramro lekh

**Comment by abhidahal** [comment]
as

**Pull request merged**